packages/workspace-store/src/navigation/helpers/traverse-document.ts

```diff
@@ -121,11 +121,11 @@ export const traverseDocument = (documentName: string, document: OpenApiDocument
       const safeIndexB = indexB === -1 ? Number.POSITIVE_INFINITY : indexB
       return safeIndexA - safeIndexB
     })
-
-    // Now update the sort order of the entries
-    document['x-scalar-order'] = unpackProxyObject(entries.map((entry) => entry.id))
   }
 
+  // Now update the sort order of the entries
```

Comment:
@DemonHa this was inside the IF before

Reply:
yea but then the order was never set!

Reply:
yea but I didn't set the order, so I didn't have access to it haha. Yea doesn't matter I could have just generated it if it didn't exist 🤷🏾